### ISSUE TITLE
c03fe1a0-c1f6-40ac-b178-865bb1e7243a

### ISSUE THREAD

**Issue by jbdaskie** [iExecBlockchainComputing/public-pool-scripts#44]
Hello
Work done (2nd time) on worker id (this time) : c03fe1a0-c1f6-40ac-b178-865bb1e7243a start at 12h34UTC, but after some time no more info on this job (Workuid : d570022c-8c10-4f52-b9ec-f56b376d1e30).

My Wallet : 0xc582c96B64EE76095AC34F518a6704CC4dA1EcC9.

Thanks for your help.

**Comment by branciard** [comment]
0xc582c96B64EE76095AC34F518a6704CC4dA1EcC9 contribute well to workorder 0x0ed5e70874487bf21ee65343fb6c5fa13382cdb6. We see contribution transaction :
https://etherscan.io/tx/0x5942fa137c96565c6297f491368ead524aaae4ebbccac1d9387449dbf3e86cc8.

Unfortunately, there is an issue in this order to be finalized : 

```

iexec work show 0x38b5c38c2e35048b36648806995d20021ed08cac --chain mainnet
ℹ iExec SDK update available 2.2.39 →  2.3.1, Run "npm -g i iexec" to update

ℹ using chain [mainnet]
✔ work 0x38b5c38c2e35048b36648806995d20021ed08cac status is ACTIVE, details:
m_workerpool:          0x0ed5e70874487bf21ee65343fb6c5fa13382cdb6
m_params:              {"cmdline":"monte-carlo normal"}
m_requester:           0x8ec966ff9752c5de46105155ef1e9a3a38542358
m_app:                 0xcb0bba8cf239e167af92a51c4212adbf9b479f6a
m_dataset:             0x0000000000000000000000000000000000000000
m_emitcost:            0
m_uri:
m_stdout:
m_resultCallbackProof: 0x0000000000000000000000000000000000000000000000000000000000000000
m_iexecHubAddress:     0x0d5ef019ca4c5cc413ee892ced89d7107c5f424d
m_callback:            0x0000000000000000000000000000000000000000
m_status:              1
m_marketorderIdx:      317
m_stderr:
m_beneficiary:         0x0000000000000000000000000000000000000000
m_statusName:          ACTIVE

```

We will manually reward on your address 0xc582c96B64EE76095AC34F518a6704CC4dA1EcC9 and indicate the transaction of the manual reward here when done.  

Same analyse for the second workorder you participate: 
```
iexec work show 0x542bfcca52177943426fd6dbc391ad68c35949af --chain mainnet
ℹ iExec SDK update available 2.2.39 →  2.3.1, Run "npm -g i iexec" to update

ℹ using chain [mainnet]
✔ work 0x542bfcca52177943426fd6dbc391ad68c35949af status is ACTIVE, details:
m_workerpool:          0x0ed5e70874487bf21ee65343fb6c5fa13382cdb6
m_params:              {"cmdline":"monte-carlo normal"}
m_requester:           0x8ec966ff9752c5de46105155ef1e9a3a38542358
m_app:                 0xcb0bba8cf239e167af92a51c4212adbf9b479f6a
m_dataset:             0x0000000000000000000000000000000000000000
m_emitcost:            0
m_uri:
m_stdout:
m_resultCallbackProof: 0x0000000000000000000000000000000000000000000000000000000000000000
m_iexecHubAddress:     0x0d5ef019ca4c5cc413ee892ced89d7107c5f424d
m_callback:            0x0000000000000000000000000000000000000000
m_status:              1
m_marketorderIdx:      399
m_stderr:
m_beneficiary:         0x0000000000000000000000000000000000000000
m_statusName:          ACTIVE

```

**Comment by branciard** [comment]
23 * 2 = 46 RLC manual reward done for worker 0xc582c96B64EE76095AC34F518a6704CC4dA1EcC9  see tx : https://etherscan.io/tx/0xddeaaa4f142b456083c53fc67c50a0f5c9020bbe4324ba1e7d333ee426f174e2
